packages/next/src/client/components/react-dev-overlay/internal/components/hot-linked-text/index.tsx

```diff
@@ -4,7 +4,7 @@ import {
   MAGIC_IDENTIFIER_REGEX,
 } from '../../../../../../shared/lib/magic-identifier'
 
-const linkRegex = /https?:\/\/[^\s/$.?#].[^\s"]*/i
+const linkRegex = /https?:\/\/[^\s/$.?#].[^\s)'"]*/i
```

Comment:
This change looks strange, what's the difference?

Reply:
@ForsakenHarmony is this intentional for sth? It’s from the 1st commit of moving folders

Reply:
This was a change done to the pages overlay, but not the app overlay before

Reply:
https://github.com/vercel/next.js/commit/05d887dc3bc31f18c3e630ea052251a63a4f202c